src/main/scala/com/particeep/api/models/transaction/TransactionSearchStatistics.scala

```diff
@@ -0,0 +1,19 @@
+package com.particeep.api.models.transaction
+
+import ai.x.play.json.Jsonx
+import ai.x.play.json.Encoders._
+import com.particeep.api.core.Formatter
+
+case class TransactionSearchStatistics(
+    amount:              Long,
+    bare_owner_amount:   Long,
+    fees:                Long,
+    usufructuary_amount: Long,
+    partner_fees:        Long,
+    num_of_shares:       Long
+)
+
+object TransactionSearchStatistics {
+  implicit val date_format = Formatter.OffsetDateTimeWrites
```

Comment:
pas de private[this] ou ça ne pourrait pas marcher ?

Reply:
ça ne marche pas :/
![image](https://github.com/Particeep/api-sdk-scala-with-play/assets/46979089/519c3407-6f95-4584-9a7e-f5c7c164d658)